include/mbedtls/ssl.h

```diff
@@ -4357,12 +4357,41 @@ int mbedtls_ssl_get_session( const mbedtls_ssl_context *ssl,
  */
 int mbedtls_ssl_handshake( mbedtls_ssl_context *ssl );
 
+/**
+ * \brief          After calling mbedtls_ssl_handshake() to start the SSL
+ *                 handshake you can call this function to check whether the
+ *                 handshake is over for a given SSL context. This function
+ *                 should be also used to determine when to stop calling
+ *                 mbedtls_handshake_step() for that context.
+ *
+ * \param ssl      SSL context
+ *
+ * \return         \c 1 if handshake is over, \c 0 if it is still ongoing.
```

Comment:
Is this only the initial handshake, or also renegotiations? Can this function return 0 during a renegotiation?